src/main/java6/net/finmath/time/Schedule.java

```diff
@@ -29,7 +29,7 @@
  */
 public class Schedule implements ScheduleInterface {
 
-	private static	DayCountConventionInterface	internalDayCounting = new DayCountConvention_ACT_365();
+	public static	DayCountConventionInterface	internalDayCounting = new DayCountConvention_ACT_365(); // static internal daycount convention to convert between doubles and date diffs
```

Comment:
I am strongly against making this field public. Reasons:

- making it public violates "Encapsulation" and "Immutability". Objects should be designed to be immutable whenever possible (not all objects in the lib are immutable, but I would like to thrive for a more functional programming paradigm). Immutablity here means: Once a schedule has been rolled out (constructed), it will never change again.
- making it public would allow for dangerous side effects (avoidance of side effect is the major reason for immutability): 
  - user creates a schedule using ACT_365
  - user then (after roll out) changes the internal day counting to ACT_360
- the internalDayCounting is **internal**. That is: it is the internal "short cut" to represent time offsets. Instead of days we use days/365 as float. There should never be a situation where this changes. 

Why is the field public?